src/pages/Details/components/Button.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import './Button.scss';
+
+const Button = props => {
+  const { color, children } = props;
```

Comment:
```suggestion
const Button = ({ color, children } ) => {
```
구조 분해 할당 위와 같이 적용해주세요~